src/main/pg/board.c

```diff
@@ -39,7 +39,9 @@ void pgResetFn_boardConfig(boardConfig_t *boardConfig)
 {
     if (boardInformationIsSet()) {
         strncpy(boardConfig->manufacturerId, getManufacturerId(), MAX_MANUFACTURER_ID_LENGTH + 1);
+        boardConfig->manufacturerId[MAX_MANUFACTURER_ID_LENGTH] = 0; // avoid GCC stringop-truncation error
```

Comment:
IMO it is safer to use sizeof - it is more resilient to refactoring.
And you don't have to copy byte that gets overwritten immediately.  
```suggestion
        strncpy(boardConfig->manufacturerId, getManufacturerId(), sizeof(boardConfig->manufacturerId));
        boardConfig->manufacturerId[sizeof(boardConfig->manufacturerId)] = 0; // ensure zero-termination of string
```